udata/frontend/helpers.py

```diff
@@ -132,25 +132,27 @@ def obfuscate(email):
 
 @front.app_template_filter()
 @front.app_template_global()
-def avatar_url(obj, size):
+@contextfilter
```

Comment:
I think we need to split in two in order to be able to use `@contextfilter` and `@contextfunction`, unless:
- both decorators works on a same function
- we try to stick to the global method or the filter everywhere